eternalcore-core/src/main/java/com/eternalcode/core/feature/teleportrequest/TeleportRequestPermissionConstant.java

```diff
@@ -0,0 +1,14 @@
+package com.eternalcode.core.feature.teleportrequest;
+
+public class TeleportRequestPermissionConstant {
```

Comment:
Nie czaję po co te constanty. To tylko utrudnia czytanie kodu

Reply:
głównie dlatego, bo `TELEPORT_TIMER_BYPASS_PERMISSION` ma wiele usage, ale pozostałe w sumie można wywalić